test/fixtures.yml

```diff
@@ -426,6 +426,11 @@ firstdata_e4_v27:
   key_id: ANINTEGER
   hmac_key: AMAGICALKEY
 
+# Working credentials, no need to replace
```

Comment:
💬 Comment:

Seems that :app_key, :app_secret, :site_id and :mid should be listed here

Reply:
Done!